src/Authorization/Services/Implementation/ContextHandler.cs

```diff
@@ -183,25 +183,34 @@ protected async Task EnrichResourceParty(XacmlContextAttributes requestResourceA
         {
             if (string.IsNullOrEmpty(resourceAttributes.ResourcePartyValue) && !string.IsNullOrEmpty(resourceAttributes.OrganizationNumber))
             {
-                int partyId = await _registerService.PartyLookup(resourceAttributes.OrganizationNumber, null);
-                if (partyId != 0)
+                Party party = await _registerService.PartyLookup(resourceAttributes.OrganizationNumber, null);
```

Comment:
For app instances this code is to early since partyID will not be set before after the instance has been retrieved

Reply:
Can we maybe delay this to later when we know AccessList is required?

Reply:
Removed the org.number lookup from contexthandler and moved it to the accesslist authorization request builder logic.